shortlister/model.py

```diff
@@ -114,4 +120,8 @@ def update_applicant_score(
 
 def update_applicant_notes(applicant: Applicant, new_note: str):
     """Appends new note to applicant's notes section."""
-    applicant.notes += f"; {new_note}"
+
+    if applicant.notes == "":
+        applicant.notes += new_note
+    else:
+        applicant.notes += f"; {new_note}"
```

Comment:
This is correct but I suggest separating the logic a bit. Under both conditions we append new note - that's common. Let's pull that bit out and make the condition wrap only the bit which is different.

```suggestion

    if len(applicant.notes):
        applicant.notes += "; "
    
    applicant.notes += new_note
```